client/js/components/user/DpUserFormFields.vue

```diff
@@ -329,6 +337,18 @@ export default {
       return new Promise((resolve, reject) => resolve(true))
     },
 
+    getOrgaAllowedRoles (orgaId) {
+      let allowedRoles = []
+      this.getOrganisations().then(() => {
```

Comment:
thx! you are right, it was not a good idea to fetch all organisations instead of just one. 

The way to save orga with it's relationships in store was a bit tricky, but it works now. You can check it here https://github.com/demos-europe/demosplan-core/pull/750/commits/2d226f1b352ea6c0e46156223290ee030695bede

Reply:
Why not use `fetchCurrentOrganisation`?

Reply:
I can use `fetchCurrentOrganisation` instead, but it will fetch `allowedRoles` by mounting even though they are already available. We get orga.allowedRoles in the `DpUserList` already (line: 229 - 235) so I use a different method when needed to fetch `allowedRoles`.

Reply:
I think ideally, we'd merge the two methods, but since it might have some undesired side effects and I don't want to delay this PR further, let's leave it as is. But could you add a comment explaining why you introduced the new method?

Reply:
done here https://github.com/demos-europe/demosplan-core/pull/750/commits/797b5e728e3fd718f589ff6e084b7b65e766650e